doc/integrator/backend_qgis.rst

```diff
@@ -134,3 +134,19 @@ like that:
    user=${PGUSER}
    password=${PGPASSWORD}
    port=${PGPORT}
+
+OGC server
+==========
+
+To use the QGIS server in authenticated mode throw the mapserv proxy, it's required to be in docker mode,
```

Comment:
What do you mean by "throw" in that context?

Reply:
Fixed :-)